src/Factory.php

```diff
@@ -11,6 +11,42 @@
  */
```

Comment:
This has an unused import for `Psr\Log\LogLevel` above (can't comment on that line though in github)